src/LettuceObject.cpp

```diff
@@ -175,10 +175,28 @@ float const LettuceObject::Rotation() const {
     return _rotation;
 }
 
+void LettuceObject::SetRotation(float rotation) {
+    if (rotation >= 360) {
```

Comment:
if statements are redundant, just go straight into the while

Reply:
you are not using degree based rotation...
![giphy](https://github.com/user-attachments/assets/41955834-a8ad-45f6-8dc8-5e9825d89ea6)

Reply:
Also `_rotation = fmod(rotation, 360);`

Reply:
ill add a radian converter for rad maxxers out there

last time i used `fmod` it had weird shenanigans with decimals near smaller values, i was also a fool/in-college last time so maybe it was user error. ill try it out again

Reply:
Food should be fine unless you are suggesting there is a floating point error in C